### ISSUE TITLE
Which training storiess were used?

### ISSUE THREAD

**Issue by GabrielKP** [GabrielKP/enc#51]
Figure out which train stories were used in Lebel et al. 2023

**Comment by KristijanArmeni** [comment]
`run_all_replication.py` currently creates a list variable `train_stories` ([around l. 56](https://github.com/GabrielKP/enc/blob/8e030854fa500fb6d4103f649a2080ffbff0694f/src/lebel_encoding/run_all_replication.py#L56)) which to my checking contains 26 total stories available for sampling. 

```python
>>> len(train_stories)
26
```
If I understand everything ok, then each repetition of N = 25 stories sampling without replacement from this pool will contain very similar training datasets (almost equal). It's not clear what dataset was used in the Lebel et al. dataset paper, but they say:

> The extended datasets are not evaluated in this paper, but see Tang et al.[50](https://www.nature.com/articles/s41597-023-02437-z#ref-CR50) for analyses showing that encoding and decoding model performance scale roughly log-linearly with the amount of training data per subject.


Our pipeline, currently (locally on the cluster) loads 80 stories of which 1 is reserved for test set. So we use a pool of `N = 79` stories for sampling training datasets.

```
from encoders.utils import load_config

stories = load_config()["STORIES"]
print(len(stories)) # prints 80
```
